cmd/gather-sysinfo/gather-sysinfo_test.go

```diff
@@ -80,13 +60,6 @@ func TestChroot(t *testing.T) {
 	}
 }
 
-func TestDeduplication(t *testing.T) {
-	resultEntries := dedupExpectedContent(kniEntries, snapshotEntries)
```

Comment:
should we have an alternative test for dedupExpectedContent ?

Reply:
Thank you for looking! Not sure it makes sense in this case. I believe it is better to focus the energy on writing correct code rather than devising unit tests that might have different requirements in the future as ghw changes/improves.  But if you insist, happy to write one.